src/main/resources/application.yml

```diff
@@ -21,3 +22,8 @@ spring:
           time_zone: UTC
   liquibase:
     enabled: false
+logging:
+  level:
+    root: INFO
+  file:
+    name: console_output.log
```

Comment:
Should this file be added in the `.gitignore` so we avoid pushing it?

Reply:
If you mean the `console.log` file, it is already disabled in the `.gitignore` file, using the wildcard `*.log`